packages/apollo-boost/package.json

```
@@ -30,9 +30,10 @@
     "build": "tsc -b .",
     "postbuild": "npm run bundle",
     "watch": "tsc -w -p .",
-    "clean": "rm -rf coverage/* lib/*",
+    "clean": "rimraf coverage/* lib/*",
+    "clean:deps": "npm run clean && rimraf node_modules/*",
     "prepublishOnly": "npm run build",
-    "bundle": "../../node_modules/rollup/bin/rollup -c rollup.config.js"
+    "bundle": "rollup -c rollup.config.js"
```

```suggestion
    "bundle": "npx rollup -c rollup.config.js"
```
If I recall correctly, using `npx rollup` means we will run the local version of `rollup` that's installed in `../../node_modules` rather than whatever copy happens to come first in the user's `PATH`.

---

if I'm not mistaken npx is used to execute a command without having to install the package (so install the component if it is not present in the PATH and delete it).

Perhaps it is better to use npx for rimraf in the deletion of the main node_modules.

---

Yes, but if it's already installed in a local `node_modules` directory (as `rollup` should be in this case), then `npx` will use the installed version without downloading anything.

---

Exactly, I didn't like the idea that if it didn't find rollup installed the command would still be executed. But if you prefer I make the change :)

---

@hwillson Do you have any thoughts on this?

---

Hi @benjamn,
some news?